contractcourt/channel_arbitrator.go

```diff
@@ -3048,3 +3055,32 @@ func (c *ChannelArbitrator) checkLegacyBreach() (ArbitratorState, error) {
 	// This is a modern breach close with resolvers.
 	return StateContractClosed, nil
 }
+
+// fetchLocalForceCloseInsight fetches the details stored about the local
+// force close previously stored in the arbitrator log and returns it as
+// LocalForceCloseInsights.
+func (c *ChannelArbitrator) fetchLocalForceCloseInsights() *channeldb.
```

Comment:
So this one actually seems to behave the way I would expect. Why do we have two functions named the same thing that do conceptually differeint things

https://github.com/lightningnetwork/lnd/pull/8072/files#r1417996512

Reply:
They are in two different packages